### ISSUE TITLE
Pandas `to_hdf` causing crash on Macs

### ISSUE THREAD

**Issue by billetd** [klaundal/lompe#48]
Found this when trying to use the AMPERE pre-processing script `read_iridium()`. It was constantly running into a client crash (python quit unexpectedly) whilst trying to save the AMPERE dataframe to a HDF5 file on [line 747](https://github.com/klaundal/lompe/blob/af7978705d8cd9bea9265d97baabfc193355cb8b/lompe/data_tools/dataloader.py#L747). The problem isn't just here though, it's when trying to save any dataframe to a HDF5 (even a very simple one), so it's not a Lompe problem.

This seems like an issue with [`pytables`](https://pypi.org/project/tables/) on Macs, possible related to the M1+ CPU architecture (my speculation, I actually have no idea). Anyway, I managed to fix the crash and get the `to_hdf5()` method to work everywhere by downgrading pytables to version 3.8:
```
pip uninstall tables
pip install tables==3.8.0
```

**Comment by klaundal** [comment]
Good that you got it to work! I haven't heard about this before, but I have intel cpu...

@FasilGibdaw did you have similar issues with your mac?

**Comment by FasilGibdaw** [comment]
yes, I had this problem, however, updating pytables was my fix, the version in my system(M3 Max) is `3.9.2`.
